distros/ubuntu1604/rules

```diff
@@ -25,7 +25,10 @@ override_dh_auto_configure:
             -DZM_SOCKDIR="/var/run/zm"                 \
             -DZM_TMPDIR="/tmp/zm"                      \
             -DZM_CGIDIR="/usr/lib/zoneminder/cgi-bin"  \
-            -DZM_CONTENTDIR="/var/cache/zoneminder"
+            -DZM_CONTENTDIR="/var/cache/zoneminder"    \
```

Comment:
I don't require anything to be done about this, but I can't find anywhere where ZM_CONTENTDIR is used.